### DISCUSSION TITLE
Vaillant Arotherm Parameter Interpretation

### DISCUSSION THREAD

**Discussion by BenKrae** [john30/ebusd#727] [Q&A] [answered]
Hello everyone,
I have a arotherm plus 105/6 installed.
Reading the parameter values works great with ebusd.
I am just struggling with the interpretation of the value "CurrentCompressorUtil".
In the ebusd-csv-documentation it says that this parameter contains the current modulation in "%".
But as you can see on the screenshot below, the value changes between around -130 and 130.
Does anyone know how to interpret this parameter and its values?
![wp_hz](https://user-images.githubusercontent.com/32748222/202996190-87c1319f-0e4c-4e5f-84e9-90f038243bc7.png)

**Comment by cyberthom42** [comment, marked as answer]
I guess your csv is wrong. Can you check that you use D2C instead of D1B. This is my line (I have an arotherm 55/6):
r,,CurrentCompressorUtil,,,,,25,,,D2C,,%,aktuelle Kompressorauslastung in Prozent,,,,,,,,,,,,,,,,,,
The statistik works great, exactly inline with the rotation speed:
![image](https://user-images.githubusercontent.com/115342533/203027582-adf28721-f609-48f0-b86f-b1f0142da96d.png)

**Comment by BenKrae** [comment]
Ok, first i am happy to see that it's working correctly for other people.
Running "ebusctl info" shows me that [this ](https://github.com/john30/ebusd-configuration/blob/master/ebusd-2.1.x/de/vaillant/08.hmu.csv)csv is loaded. 
So I am indeed using D1B.
Could you please point me in the right direction:
- What does D1B and D2C mean ?
- How do I change the value? I guess I need to download the csv, change the value and import it again somehow.
